package.json

```diff
@@ -12,6 +12,7 @@
     "networks-extract": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/extract_network_info.js",
     "networks-inject": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/inject_network_info.js",
     "networks-reset": "mkdir -p build/contracts && npx truffle networks --clean && npm run networks-inject",
+    "verify-stablex-rinkeby": "npx truffle run verify StablecoinConverter --network rinkeby",
```

Comment:
If we define the script as:
`"verify-stablex": "npx truffle run verify StablecoinConverter",`

We can use it for any network:
```bash
npm run verify-stablex --network rinkeby
npm run verify-stablex --network mainnet
```